src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,74 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int STORAGE_SIZE = 10;
+    private K[] keyDatabase;
+    private V[] valueDatabase;
```

Comment:
We don't use the database
```suggestion
    private K[] keys;
    private V[] values;
```